src/native/minipal/debugger.c

```diff
@@ -129,6 +131,14 @@ bool minipal_is_native_debugger_present(void)
     getprocs64(&proc, sizeof(proc), NULL, 0, &pid, 1);
     return (proc.pi_flags & STRC) != 0; // SMPTRACE or SWTED might work too
 
+#elif defined(__HAIKU__)
+    team_info info;
+    if (get_team_info(B_CURRENT_TEAM, &info) == B_OK)
+    {
+        return info.debugger_nub_thread > 0;
+    }
+    return FALSE;
```

Comment:
I am sorry for missing it before, this should return "false" instead of "FALSE"

Reply:
Done.

I overlooked that, thought it was just a simple copy-paste...